vtr_flow/arch/COFFE_22nm/k6FracN10LB_mem20K_complexDSP_customSB_22nm.clustered.denser.xml

```diff
@@ -0,0 +1,3159 @@
+<!--
+    This is the architecture file for an Agilex-like Architecture, based off
```

Comment:
I think we should qualify this just like we did in the paper. The blocks are Agilex-like (logic, RAM, DSP), but the routing is not -- it is a port of the Stratix IV like architecture.
Same comment for other arch files.